dotCMS/src/main/java/com/dotmarketing/portlets/containers/model/FileAssetContainer.java

```diff
@@ -42,15 +42,28 @@ public FileAssetContainer() {
     @JsonIgnore
     private transient FileAsset postLoopAsset = null;
 
+    @JsonIgnore
+    private transient FileAsset defaultContainerLayoutAsset = null;
```

Comment:
Codacy found an issue: [Avoid using redundant field initializer for 'defaultContainerLayoutAsset'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6347242)